.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   extends: ['vinta/recommended'],
   rules: {
     "default-param-last": "off",
+    "@babel/camelcase": "off"
```

Comment:
Apparently, this rule is deprecated. It doesn't point anything for us to change, but still raises errors. It must because of some outdated eslint dependency that is pinned by vinta's plugin